include/kernels/dslash_wilson.cuh

```diff
@@ -52,11 +52,7 @@ namespace quda
       checkLocation(out, in, x, U);  // check all locations match
       if (!in.isNative() || !U.isNative())
         errorQuda("Unsupported field order colorspinor=%d gauge=%d combination\n", in.FieldOrder(), U.FieldOrder());
-
-      if (F::N != F::N_ghost) pushKernelPackT(true); // must use packing kernel is ghost vector length is different than bulk
```

Comment:
I might find that later but just wondering why this is gone

Reply:
This is gone because we now always do kernel packing.